### PR TITLE
Redesign

### LINKED ISSUE #312: 🇸🇻 Multi-language support
I think this is a great opportunity to add multi language support after the refactoring is done.  We don’t have to translate everything right away but having the language files ready for integration would be huge.  I'm sure the community will help us get Osmosis translated as @jonator  mentioned. 

Maybe: 
https://www.npmjs.com/package/react-multi-lang

 @dogemos this would be a great item to add to Keplr’s backlog too.  I think this is really important for mass adoption.

### LINKED ISSUE #709: Handle dusted assets (pools)
[From discord](https://discord.com/channels/798583171548840026/855908784445849610/1002841988858654730) and given recent dusting spam.

Similar to how we filter pools < $1000 TVL, we should look into filtering pools or asset balances with dust amounts.

Things to consider:
* What constitutes dust? A min fiat value?
* How to add this feature in harmony with existing <1000TVL filter. A dropdown of filter toggles?

### LINKED ISSUE #831: Redesign pools page
[Figma](https://www.figma.com/file/ddlsIdRm0p30bFC5Lyt5OA/Pools?node-id=115%3A3777)

### LINKED ISSUE #832: Redesign assets page
[Figma](https://www.figma.com/file/TbhltqFzPjWgIdxdcQ23dj/Assets?node-id=46%3A811)

### LINKED ISSUE #833: Redesign pool detail page
[Working Design Figma](https://www.figma.com/file/7iWExDLY1u7kClXZwS3OZI/Pool-Details?node-id=161%3A4289)

### LINKED ISSUE #894: New amplitude events
Work with @victortokaito and @delivan on adding new amplitude events.

### LINKED ISSUE #895: New mobile design

### LINKED ISSUE #901: Integrate Kado
May be as simple as opening an iframe with:
```
https://app.kado.money/?onPayCurrency=USD&onPayAmount=200&onRevCurrency=USDC&offPayCurrency=USDC&offRevCurrency=USD&network=OSMOSIS&onToAddress=X&offFromAddress=X&cryptoList=USDC&networkList=OSMOSIS&apiKey=67a2aaad-a2a5-4412-be40-e3a70aa7b53d&product=BUY
```

### LINKED ISSUE #902: Integrate Transak

### LINKED ISSUE #919: Update localizations to new copy, add Korean
- [x] @DLatouche update FR (will get you a diff of what changed since your PR, mostly on pool detail page)
- [x] @dogemos or @delivan  add/delegate KR

### LINKED ISSUE #922: Update OGP Meta image
Download this image and set it in the OGP Meta component. This is used in previews when sharing links on socials, iMessage, Slack, etc.

preview image (1)

### LINKED ISSUE #928: Redesign bug bash

### LINKED ISSUE #936: Clean up pre-redesign code
We shouldn't keep around dead code:
- [ ] Tailwind config
- [ ] Components/hooks

### LINKED ISSUE #959: Add Polish translation
According to [this](https://github.com/osmosis-labs/osmosis-frontend/issues/312#issuecomment-1216942779), 4% of osmosis users speak Polish, and it's a 5th largest group. 

I think it makes sense for it to be one of the first available translations on the site after the redesign.

### PR DESCRIPTION
**Pull request by jonator**
* Closes #901 
* Closes #902 
* Closes #831 
* Closes #832 
* Closes #709 
* Closes #833 
* Closes #894 
* Closes #895 
* Closes #919 
* Closes #928 
* Closes #936
* Closes #922 
* Closes #928 
* Closes #312
* Closes #959
- [x] Add back link from PR #954

### CODE REVIEW

**Comment by gnujoow** [comment]
@jonator
I've checked the pr but I couldn't find the JSON file related to the Korean translation. If it's not going on yet, can I contribute about translation? #919

**Comment by jonator** [comment]
@gnujoow yeah you'd just create a json file that has the same structure as the others. Then a new menu option for Korean. @victortokaito may know of a korean flag icon to use.

**Comment by gnujoow** [comment]
> 

thanks for replying, I will make pr asap.

File: packages/web/localizations/fr.json [hunk 1]
```diff
@@ -215,6 +215,30 @@
...
     },
+    "transfer": {
+      "titleWithdraw": "Retirer {coinDenom}",
+      "titleDeposit": "Déposer {coinDenom}",
+      "warnDepositAddressDifferent": "Attention : le compte sélectionné dans {address} diffère du compte avec lequel vous avez effectué le dernier dépôt.",
+      "from": "De",
+      "to": "À",
+      "via": "via",
+      "loading": "Chargement",
+      "availableOn": "Disponible sur  {network}",
```
Comment (DLatouche) [inline packages/web/localizations/fr.json:226]: I think there is an extra space

**Comment by jonator** [comment]
@gnujoow I think you're good to update translations. I'm confident I've found all the translations we'd need, including errors.

File: packages/stores/src/queries-external/store.ts [hunk 2]
```diff
@@ -1,41 +1,28 @@
...
+export class QueriesExternalStore {
   public readonly queryGammPoolFeeMetrics: DeepReadonly<ObservableQueryPoolFeesMetrics>;
+  public readonly queryAccountsPoolRewards: DeepReadonly<ObservableQueryAccountsPoolRewards>;
 
   constructor(
     kvStore: KVStore,
-    feeMetricsBaseURL = "https://api-osmosis.imperator.co"
+    priceStore: IPriceStore,
+    feeMetricsBaseURL = "https://api-osmosis.imperator.co",
+    poolRewardsBaseUrl = "https://api-osmosis-chain.imperator.co"
```
Comment (delivan) [inline packages/stores/src/queries-external/store.ts:16]: How about making these values constant? I've seen these value used in other files

File: packages/web/components/complex/compact-pool-table-display.tsx [hunk 3]
```diff
@@ -39,35 +36,31 @@ export const CompactPoolTableDisplay: FunctionComponent<{
 }) => {
   const router = useRouter();
 
+  console.log(sortMenuProps?.options);
```
Comment (delivan) [inline packages/web/components/complex/compact-pool-table-display.tsx:39]: Remove console

File: packages/web/components/complex/pool/create/step1-set-ratios.tsx [hunk 4]
```diff
@@ -7,63 +7,68 @@ import { InputBox } from "../../../input";
...
                 onSelect={(coinDenom) => {
                   const currency = config.remainingSelectableCurrencies.find(
                     (currency) => currency.coinDenom === coinDenom
                   );
                   if (currency) {
                     amountConfig.setSendCurrency(currency);
+                  } else {
+                    console.error(
+                      "Unable to find currency selected in TokenSelect to be set in create pool config"
+                    );
```
Comment (delivan) [inline packages/web/components/complex/pool/create/step1-set-ratios.tsx:41]: Do we need to make this an error object too?

Comment (jonator) [reply packages/web/components/complex/pool/create/step1-set-ratios.tsx:41]: What do you mean by error object? It's basically to help debug.

File: packages/stores/src/queries-external/store.ts (hunk 2 above)
Comment (delivan) [reply packages/stores/src/queries-external/store.ts:16]: Yes, const variable that can be shared

File: packages/web/components/complex/pool/create/step1-set-ratios.tsx (hunk 4 above)
Comment (delivan) [reply packages/web/components/complex/pool/create/step1-set-ratios.tsx:41]: like this you made on `packages/stores/src/ui-config/errors.ts`
```ts
export class NegativeSwapFeeError extends Error {
  constructor(m: string) {
    super(m);
    // Set the prototype explicitly.
    Object.setPrototypeOf(this, NegativeSwapFeeError.prototype);
  }
}
```

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 6 42 21 PM
There should be a space between the dropdown and options
Screen Shot 2022-11-07 at 6 43 21 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 6 45 38 PM
This button seems a bit tall.

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 6 49 45 PM
We need properties for these inner cards

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 10 45 58 PM
Spacing seems off. Double check Figma as seen below: 
Screen Shot 2022-11-07 at 10 47 03 PM

**Comment by syed8rahman** [comment]
Should be much more padding between the bonded + unbonded stats and your total value 

Figma:
Screen Shot 2022-11-07 at 10 48 49 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 10 51 32 PM
Why is the superfluid toggle not interactive? It's stuck in a disabled state.

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 10 53 52 PM
hover states are missing.

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 10 56 15 PM
Minor padding issue with the wallet button: should be padding of 8 px horizontal.

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 10 58 39 PM

Settings should open up with respect to the settings button as shown below
Screen Shot 2022-11-07 at 10 59 13 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 00 08 PM
Language select button is tall.

Should be 40px in height.

Screen Shot 2022-11-07 at 11 00 50 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 01 40 PM
Can we dim the background image (hexes) by half?

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 03 00 PM
Assets dash needs to be Osmosis background 1000. 
Screen Shot 2022-11-07 at 11 03 56 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 04 28 PM
Button widths should be 180px
Screen Shot 2022-11-07 at 11 04 48 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 05 35 PM
Is spacing between buttons 12px? 
Figma reference below is different: 
Screen Shot 2022-11-07 at 11 05 51 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 06 42 PM
Let's expand this so Remove liquidity can fit on one line

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 07 45 PM
I think we're missing hover states for all text based buttons. 

Here is the reference in design guidelines.
Screen Shot 2022-11-07 at 11 08 02 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-07 at 11 08 42 PM
We shouldn't have an empty space if there's nothing to withdraw, we need a disabled withdraw button state. 
Just added this to Ready for dev in Assets page. 

Screen Shot 2022-11-07 at 11 10 03 PM

**Comment by jonator** [comment]
> Screen Shot 2022-11-07 at 10 47 29 PM
> 
> Are text properties correct on buttons? The font seems quite large
> Figma reference: Screen Shot 2022-11-07 at 10 47 58 PM

Yeah it's the same; `h6` as it is in Figma. The weight, size, and line spacing are the same as well

**Comment by jonator** [comment]
> Screen Shot 2022-11-07 at 11 08 42 PM
> 
> We shouldn't have an empty space if there's nothing to withdraw, we need a disabled withdraw button state. Just added this to Ready for dev in Assets page.
> Screen Shot 2022-11-07 at 11 10 03 PM

Are you referring to the OSMO row? It's a special case as it isn't really deposited or withdrawn. However, buying it could be thought of depositing from bank account.

**Comment by syed8rahman** [comment]
> > Screen Shot 2022-11-07 at 10 47 29 PM
> > Are text properties correct on buttons? The font seems quite large
> > Figma reference: Screen Shot 2022-11-07 at 10 47 58 PM
> 
> Yeah it's the same; `h6` as it is in Figma. The weight, size, and line spacing are the same as well

Hmmm there's definitely some sort of discrepancy here. Just double checking H6 specs - dropped a screenshot below.
Screen Shot 2022-11-08 at 12 09 11 AM

**Comment by jonator** [comment]
> Screen Shot 2022-11-07 at 10 51 32 PM
> 
> Why is the superfluid toggle not interactive? It's stuck in a disabled state.

This is because the user already selected an SFS validator for that pool. It will be reused on future bonds to the 14 day duration.

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-08 at 5 52 31 PM
Row contents should be aligned on the left with the column title. It's a bit far left. 

Figma below. 
Screen Shot 2022-11-08 at 5 52 56 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-08 at 5 58 38 PM
Nav bar items start pretty low - should start higher up.

Figma
Screen Shot 2022-11-08 at 5 58 59 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-08 at 6 01 42 PM
It looks like the deposit withdraw buttons are different heights - might be a positioning of stroke? Inner vs. outer. 

Figma
Screen Shot 2022-11-08 at 6 02 05 PM

**Comment by syed8rahman** [comment]
Screen Shot 2022-11-08 at 6 40 37 PM
Too much padding between search + filter

File: packages/web/components/main-menu.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                {
+                  "rounded-full bg-wosmongton-500": selected,
+                }
+              )}
+              onClick={
+                typeof link === "string" ? () => router.push(link) : link
+              }
+            >
+              <Head>{selected && <title key="title">{label}</title>}</Head>
+              <LinkOrDiv href={link}>
```
Comment (delivan) [inline packages/web/components/main-menu.tsx:42]: opening a link in a new tab doesn'work. It seems to need to remove `Link` component if a menu has external link.

**Review by delivan — APPROVED** [review]
LGTM👍

**Comment by jonator** [comment]
Github is bugged out. It merged.

**Pull request closed without merging**